_config.yml

```diff
@@ -80,10 +80,18 @@ footer:
     # Version info of NexT after scheme info (vX.X.X).
     version: true
 
-  # Beian ICP information for Chinese users. See: http://www.beian.miit.gov.cn
+  # Beian ICP and gongan information for Chinese users.
+  # See: http://www.beian.miit.gov.cn
+  # See: http://www.beian.gov.cn
   beian:
     enable: false
     icp:
+    # The digit in th num of gongan beian.
```

Comment:
Do you mean `the`

Reply:
Yeah. I typed wrongly. It should be the.